### PR TITLE
Rick and loc cookie9

### PR DESCRIPTION
**Pull request by Rick-Anderson**

### CODE REVIEW

**Comment by Rick-Anderson** [comment]
@antonGritsenko please review

**Comment by Rick-Anderson** [comment]
@ryanbrandenburg  please review

File: aspnet/fundamentals/localization.rst [hunk 1]
```diff
@@ -203,7 +203,16 @@ If you only pass in one of the two (``culture`` or ``ui-culture``), the query st
 CookieRequestCultureProvider
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Production apps will often provide a mechanism to set the culture with the ``ASPNET_CULTURE`` cookie. The sample app provides code to set the cookie for French and English cultures.
+Production apps will often provide a mechanism to set the culture with the ASP.NET Core culture cookie. Use the :dn:method:`~Microsoft.AspNetCore.Localization.CookieRequestCultureProvider.MakeCookieValue`  method of the :dn:cls:`~Microsoft.AspNetCore.Localization.CookieRequestCultureProvider` to create a cookie.
```
Comment (ryanbrandenburg) [inline aspnet/fundamentals/localization.rst]: Is this `:dn:` stuff the normal format? It doesn't show up at least anywhere else in this file is why I ask.

Also, just a nitpick, but I would normally phrase this as "Use the `blah` method to create a cookie.". The namespace it's in is already available from that link right?

**Comment by danroth27** [comment]
Looks good to me :shipit:

**Comment by ryanbrandenburg** [comment]
:shipit: other than formatting nitpicks that @danroth27 explained the reasoning for.

**Comment by Rick-Anderson** [comment]
@ryanbrandenburg thanks - you're right namespace not needed. I removed that and it reads better now.

**Comment by antonGritsenko** [comment]
Much cleaner, thanks.
But dn: staff looks wrong, isn't it?

**Comment by danroth27** [comment]
The :dn: stuff is our new Sphinx .NET domain support for referencing the API links. We now have semantic linking to types and members.

**Comment by antonGritsenko** [comment]
Well, in any case links doesn't work.

**Comment by danroth27** [comment]
GitHub won't know how to render these links, but they should show up in the actual doc builds.

I just tried out the links on the live site (https://docs.asp.net/en/latest/fundamentals/localization.html#cookierequestcultureprovider) and they seem to be working. Are you seeing something different?

**Comment by antonGritsenko** [comment]
Aha, gotcha! Thanks for clarification. Much appreciated.

**Pull request merged**